osu.Game/Storyboards/Drawables/DrawableStoryboardAnimation.cs

```diff
@@ -17,6 +17,8 @@ public class DrawableStoryboardAnimation : TextureAnimation, IFlippable
         public bool FlipH { get; set; }
         public bool FlipV { get; set; }
 
+        public override bool RemoveWhenNotAlive => false;

```

Comment:
Do we need this here? This should recurse down from DrawableStoryboard.

Reply:
Is that how it works? I don't think this is even set in `DrawableStoryboard`..